swift/src/Ice/OutputStream.swift

```diff
@@ -134,7 +132,7 @@ public class OutputStream {
     public func writePendingValues() {
         if encaps != nil, encaps.encoder != nil {
             encaps.encoder.writePendingValues()
-        } else if encoding_1_0 {
+        } else if currentEncoding == Encoding_1_0 {
```

Comment:
When is `encoding != currentEncoding`?

Reply:
If you look at startReplyStream, we construct the OutputStream with:
```swift
let ostr = OutputStream(communicator: adapter.getCommunicator(), encoding: currentProtocolEncoding)
```
i.e. 1.0. And then we later change the "current encoding" to 1.1 when we start the encapsulation.

While correct - and identical in C++ and C# - this encoding switch is not strictly necessary since what we encode in the header has the same encoding with 1.0 and 1.1. We could simplify in all languages but then it could get confusing.